### DISCUSSION TITLE
Missing WS2812 option under device configuration

### DISCUSSION THREAD

**Discussion by T94T** [arendst/Tasmota#15526] [Q&A] [answered]
I'm compiling the `tasmota32-ir` env in the PlatformIO. Compared to the `tasmota32` env there is the WS2812 ping configuration option missing.
What I tried is to add the following to the `user_config_override.h`, but without luck:
```
#define USE_LIGHT
#define USE_WS2812
```
Can anybody please point me to the right direction?

**Comment by barbudor** [comment, marked as answer]
You can't add/remove features to a tasmota variant such as tasmota32-ir, tasmota32-lvgl etc ....
You must restart from tasmota32 and add everything you need from there which means adding  IR to tasmota32 instead of adding WS2812 to tasmota32-ir

As per the docs: https://tasmota.github.io/docs/Compile-your-build/#customize-your-build

![image](https://user-images.githubusercontent.com/2996042/166555925-e9a9a068-e2e3-4d16-880d-269a7a403093.png)

**Reply by T94T** [reply]
Thanks. Now that explains a lot. I will make my configs based on tasmota32.
